sdks/python/apache_beam/testing/synthetic_pipeline_test.py

```diff
@@ -57,29 +57,39 @@ class SyntheticPipelineTest(unittest.TestCase):
 
   # pylint: disable=expression-not-assigned
 
-  def test_synthetic_step(self):
-    start = time.time()
+  def test_count_elements_after_synthetic_step(self):
     with beam.Pipeline() as p:
       pcoll = p | beam.Create(list(range(10))) | beam.ParDo(
           synthetic_pipeline.SyntheticStep(0, 0.5, 10))
       assert_that(
           pcoll | beam.combiners.Count.Globally(), equal_to([100]))
 
+  def test_minimal_runtime_with_synthetic_step_delay(self):
```

Comment:
@chamikaramj I also decided to split it into two separate test cases to be give more meaningful names (hope it's meaningful enough).